core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -565,7 +565,8 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     private PingResponse createPingResponse(DiscoveryNodes discoNodes) {
-        return new PingResponse(discoNodes.getLocalNode(), discoNodes.getMasterNode(), clusterName, contextProvider.nodeHasJoinedClusterOnce());
+        return new PingResponse(discoNodes.getLocalNode(), discoNodes.getMasterNode(), clusterName,
```

Comment:
we settled on the `get/is/set` when there is duplication we remove the one without the prefix. We had this discussion so often. The majority of the java worlds does it that way, we settled on it and moving towards it. It's like tabs/spaces - I make a call to solve the issue